lib/v2/metacritic/index.js

```diff
@@ -0,0 +1,121 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const { sorts, types } = require('./util');
+
+module.exports = async (ctx) => {
+    const { type = 'game', sort = 'new', filter } = ctx.params;
+    const limit = ctx.query.limit ? parseInt(ctx.query.limit, 10) : 50;
+
+    const rootUrl = 'https://www.metacritic.com';
+    const rootApiUrl = 'https://fandom-prod.apigee.net';
+    const apiUrl = new URL('v1/xapi/finder/metacritic/web', rootApiUrl).href;
+
+    const currentUrlObject = new URL(`/browse/${type}/all/all/all-time/${sort}${filter ? `?${filter}` : ''}`, rootUrl);
```

Comment:
```suggestion
    const currentUrlObject = new URL(`/browse/${type}/all/all/all-time/${sort}/${filter ? `?${filter}` : ''}`, rootUrl);
```
https://github.com/DIYgod/RSSHub/actions/runs/6371803693/job/17294067386#step:12:17